Ginger/Ginger/Reports/HTMLReportTemplatePage.xaml

```diff
@@ -111,6 +111,15 @@
                                         <RadioButton x:Name="htmlShowFirstIterationOnRadioBtn" Content="Yes" Padding="5,0,0,0" Checked="htmlShowFirstIterationOnRadioBtn_Checked" Style="{StaticResource @InputRadioButtonStyle}"/>
                                     </StackPanel>
                                 </StackPanel>
+                                <StackPanel x:Name="htmlExecutionCalculationLevel" Margin="10,10,0,0" Orientation="Horizontal" ToolTip="Show all iterations execution results on HTML Report e.g. Flow control or retry mechanism iterations">
```

Comment:
name of controls in xaml always should start with "x" and end with control type, like this should be "xExecutionCalculationLevelPnl"

please handle for all added controls in xaml